tests/boot/boot_from_pxe.pm

```diff
@@ -109,10 +122,26 @@ sub run {
     save_screenshot;
 
     if ((check_var('BACKEND', 'ipmi') && !get_var('AUTOYAST')) || get_var('SES5_DEPLOY')) {
-        my $ssh_vnc_wait_time = 600;
-        assert_screen((check_var('VIDEOMODE', 'text') ? 'sshd' : 'vnc') . '-server-started', $ssh_vnc_wait_time);
-        select_console 'installation';
+        my $ssh_vnc_wait_time = get_var('SES5_DEPLOY') ? 300 : 210;
+        my $ssh_vnc_tag = eval { check_var('VIDEOMODE', 'text') ? 'sshd' : 'vnc' } . '-server-started';
+        my @tags = ($ssh_vnc_tag, 'orthos-grub-boot-continue');
+        assert_screen \@tags, $ssh_vnc_wait_time;
 
+        if (match_has_tag("orthos-grub-boot-continue")) {
+            my $image_name = eval { check_var("INSTALL_TO_OTHERS", 1) ? get_var("REPO_0_TO_INSTALL") : get_var("REPO_0") };
+            my $arch       = get_var('ARCH');
+            my $args       = "initrd auto/openqa/repo/${image_name}/boot/${arch}/initrd";
+            type_string $args;
+            send_key 'ret';
+            save_screenshot;
```

Comment:
This looks like an unsafe way of synchronising the test flow. I guess a safer way is an assert_screen to make it explicit

Reply:
If you have strong feeling that this should also be made explicitly, I can also make the change. Actually, the similar screen check is also made after loading "linux", which is used as to make sure network connection is ready and synchronize test flow. The snippet you mentioned already falls into "orthos boot" category, so the "boot" action following is very definite. But for sure and unanimity, I can change to use assert_screen instead.